src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -109,7 +109,7 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
         if (!currency || !_.isNumber(amount)) {
             return;
         }
-        const amountAsStringForState = CurrencyUtils.convertToFrontendAmount(amount).toString();
+        const amountAsStringForState = amount ? CurrencyUtils.convertToFrontendAmount(amount).toString() : '';
```

Comment:
Isn't this equivalent to `!== 0`? I think that would be preferrable

Reply:
> I think that would be preferrable

I meant that if this is the case, the code should be changed to `!== 0`; or, otherwise, to:

```js
const amountAsStringForState = amount === 0 ? '' : CurrencyUtils.convertToFrontendAmount(amount).toString();
```

Whichever you prefer